app/views/users/index.html.slim

```diff
@@ -0,0 +1,25 @@
+.container.mt-3
+  h2
+    = t ".title"
+  p
+    = t ".mess"
+  input#myInput.form-control[type="text" placeholder="Search.."]
+  br
+  table.table.table-striped.table-condensed
+    thead
+      tr
+        th
+          = t ".name"
+        th
+          = t ".email"
+        th
+          = t ".role"
+    tbody#myTable
+      - (@users - [current_user]).each do |user|
```

Comment:
có thể: User.where.not(id: current_user.id) ở controllers

Reply:
fixed